shared/wallets/send-form/body/index.js

```diff
@@ -0,0 +1,35 @@
+// @flow
+import * as React from 'react'
+import {Divider, Box2} from '../../../common-adapters'
+import AssetInput from '../asset-input'
+import Banner from '../banner/container'
+import Footer from '../footer/container'
+import Memo from '../memo/container'
+import Note from '../note/container'
+import Participants from '../participants/container'
+
+type Props = {
+  bannerInfo?: string,
+  isProcessing?: boolean,
+  onClick: Function,
+}
+
+const Spinner = () => <Box2 direction="vertical">Spinner</Box2>
```

Comment:
i think we can make this spinner actually the correct implementation. just an overlay w/ the spinner in it over the correct components

Reply:
Is the implementation in `teams/really-leave-team` a good example?

Reply:
sure, it'll just be a box w/ a progressindicator inside (https://zpl.io/awrJJOg)
make sure its below the header